test/test_type_promotion.py

```diff
@@ -932,7 +932,7 @@ def test_integer_addcdiv_deprecated(self, device, dtype):
         with self.assertRaisesRegex(RuntimeError, '^Integer division.+is no longer supported+'):
             t.addcdiv_(t, t)
 
-    def _ternary_promotion_common(self, device, op1, op2):
+    def _ternary_promotion_common(self, device, op1, op2, op3):
```

Comment:
yes, i think so. i think it's not too costly to have this while it might still catch some things. it can be removed later if it's a problem. if you feel strongly about it, it can be removed when the addcmul ref lands.

Reply:
what are the "some things" that this may catch that the OpInfo may not?

Reply:
it's just that opinfo/sample/ref-inputs have too many moving bits while this is a pretty transparent test for a concrete issue. but i guess you're right, i'm going to remove it